REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.6
-Compat 0.9.5
+Compat 0.28.0
```

Comment:
as long as you replace it before travis starts running on the open metadata pr, yeah